README.md

```diff
@@ -51,6 +51,8 @@ The main objectives of the extension are:
 2) record all necessary bands, parameters, modeling artifact locations, and high-level processing steps to deploy
    an inference service.
 
+<img src="./STAC_MLM.png" width="1000px" alt="STAC_MLM">
```

Comment:
Do not enforce some width. Doesn't work nicely with other devices that your standard monitor. Simply use a markdown image reference and let it auto-scale. 

Also, I think it makes more sense to place it just before the "See Best Practices..."  below. Here, it cuts the summary description in two. 

Consider adding the citation as well under it. Just another way to communicate the paper reference along #32. 
> Francis Charette-Migneault, Ryan Avery, Brian Pondi, Joses Omojola, Simone Vaccari, Parham Membari, Devis Peressutti, Jia Yu, and Jed Sundwall. 2024. Machine Learning Model Specification for Cataloging Spatio-Temporal Models (Demo Paper). In 3rd ACM SIGSPATIAL International Workshop on Searching and Mining Large Collections of Geospatial Data (GeoSearch’24), October 29–November 1 2024, Atlanta, GA, USA. ACM, New York, NY, USA, 4 pages. https://doi.org/10.1145/3681769.3698586

Reply:
thanks, done!